Scripts/RemediationScripts/Remediate-EnableMicrosoftDefenders.ps1

```diff
@@ -0,0 +1,877 @@
+<##########################################
+
+# Overview:
+    This script is used to configure Azure Defender on subscription.
+
+# ControlId: 
+    Azure_Subscription_Config_Enable_MicrosoftDefender_Databases
+    Azure_Subscription_Config_Enable_MicrosoftDefender_ResourceManager
+    Azure_Subscription_Config_Enable_MicrosoftDefender_AppService
+    Azure_Subscription_Config_Enable_MicrosoftDefender_Storage
+    Azure_Subscription_Config_Enable_MicrosoftDefender_Container
+    Azure_Subscription_Config_Enable_MicrosoftDefender_Servers
+    Azure_Subscription_Config_Enable_MicrosoftDefender_KeyVault
+
+# Pre-requisites:
+    1. You will need Owner or Contributor role on subscription.
```

Comment:
will contri role allow to make these secure changes?

Reply:
Removed.